CRM/Admin/Form/Options.php

```diff
@@ -376,6 +376,19 @@ public static function formRule($fields, $files, $self) {
       }
     }
 
+    $optionGroup = civicrm_api3('OptionGroup', 'get', array(
```

Comment:
I would wrap this in a private method to make the code cleaner and more readable (kind of pseudo-code following):
```php
/**
 *
 *
 */
private function getOptionGroupDataType($groupName) {
  $optionGroup = civicrm_api3('OptionGroup', 'get', array(
    'sequential' => 1,
    'name' => $groupName,
  ));

  return CRM_Core_BAO_OptionGroup::getDataType($optionGroup['id']);
}

// ...

public static function formRule($fields, $files, $self) {
  $dataType = $self->getOptionGroupDataType($self->_gName);

  if ($dataType) {
    // ...
  }

 //..
}
```